src/api/users/src/routes/users.js

```diff
@@ -28,27 +28,35 @@ router.get('/:id', validateId(), async (req, res, next) => {
 });
 
 // get all users
-// rejected if the user collection is empty, otherwise users returned
-router.get('/', async (req, res, next) => {
-  try {
-    const usersRef = db.collection('users');
-    const snapshot = await usersRef.get();
+// rejected if the users collection is empty
+// otherwise n (specified via params) users are returned
+router.get('/', validatePagingParams(), async (req, res, next) => {
+  // if req.query.per_page is < 1 or unparsable, set to 100, otherwise set to the min of either req.query.page or 100
```

Comment:
This comment still doesn't reflect what the code is doing (100 vs. 20).  Either fix or remove it.